pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -87,9 +90,6 @@ let self = toPythonModule (python.stdenv.mkDerivation (builtins.removeAttrs attr
   # Propagate python and setuptools. We should stop propagating setuptools.
   propagatedBuildInputs = propagatedBuildInputs ++ [ python setuptools ];
 
-  # Enabled to detect some (native)BuildInputs mistakes
-  strictDeps = true;
```

Comment:
You still need inherit here for the default to propagate. "attrs" doesn't have defaults iirc

Reply:
Hmm, you are right:

```ShellSession
$ nix-instantiate --eval -E '(({ test ? true } @ args: args) {})'
{ }
```